### PR TITLE
User pages + Database Cleaner implementation

### PR DESCRIPTION
**Pull request by marysalemme**
💃

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+  def show
+    if User.exists?(params[:id])
+      @user = User.find(params[:id])
+      p @user.id
```
Comment (alistairkung) [inline app/controllers/users_controller.rb]: can we remove the p on line 5?

Comment (tbscanlon) [inline app/controllers/users_controller.rb]: if the model is working, do we need to p anything?

**Review by alistairkung — CHANGES_REQUESTED** [review]
:gun:

File: spec/helpers/users_helper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# require 'rails_helper'
```
Comment (vhonebon1) [inline spec/helpers/users_helper_spec.rb:1]: This is commented out.  what do we want to do about these types of test?

**Review by vhonebon1 — COMMENTED** [review]
What are we going to do about rails generated tests that start as pending?  There is a test commented out in this request.

File: spec/controllers/users_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe "GET /show " do
+    it "responds with 200" do
+      get :show
+      expect(response).to have_http_status(200)
+    end
+    it "adds user to database" do
+      expect{User.create}.to change{User.count}.by(1)
```
Comment (tbscanlon) [inline spec/controllers/users_controller_spec.rb]: Can we add spaces around the curly braces, as per the Ruby Style Guide?
i.e. `expect { User.create }.to change { User.count } `

**Review by tbscanlon — CHANGES_REQUESTED** [review]

**Review by mihobo — COMMENTED** [review]
Removed a p, edited in spaces for Ruby conventions

**Pull request closed without merging**